ts/nni_manager/training_service/local_v3/local.ts

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Logger, getLogger } from 'common/log';
+import type { LocalConfig, TrainingServiceConfig } from 'common/experimentConfig';
+import type { EnvironmentInfo, Metric, Parameter, TrainingServiceV3 } from 'common/training_service_v3';
+import { TrialKeeper } from 'common/trial_keeper/keeper';
+
+export class LocalTrainingServiceV3 implements TrainingServiceV3 {
+    private config: LocalConfig;
+    private env: EnvironmentInfo;
+    private log: Logger;
+    private trialKeeper: TrialKeeper;
+
+    constructor(trainingServiceId: string, config: TrainingServiceConfig) {
+        this.log = getLogger(`LocalV3.${trainingServiceId}`);
+        this.log.debug('Training sevice config:', config);
+
+        this.config = config as LocalConfig;
+        this.env = { id: `${trainingServiceId}-env` };
+        this.trialKeeper = new TrialKeeper(this.env.id, 'local', Boolean(config.trialGpuNumber));
```

Comment:
`trialKeeper` is only for local training service, right? because it is in typescript.